vmdb/app/models/provider.rb

```diff
@@ -0,0 +1,18 @@
+class Provider < ActiveRecord::Base
+  include AuthenticationMixin
+
+  belongs_to :zone
+
+  def verify_ssl=(val)
+    val = case val
+          when true then OpenSSL::SSL::VERIFY_PEER
+          when false then OpenSSL::SSL::VERIFY_NONE
```

Comment:
Would it make sense to store the a string or the string constant name in the database instead of the integer?  At least when you look at a database you would see the string and know the value instead of just seeing "0" and have to go figure out what that means.